### PR TITLE
Create PRS in BLorient11602

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I have not found person with the initials J. St. Lowe. He was perhaps a relative to PRS14317Rowe, but I was not able to verify that.

### CODE REVIEW

**Review by karljonaskarlsson — APPROVED** [review]

**Comment by karljonaskarlsson** [comment]
How come it's already possible to find the first person in the app (https://betamasaheft.eu/persons/PRS14536JLowe/main?hi=), even though the pull request has not been merged?

**Comment by eu-genia** [comment]
As I had explained, when a record is created in the app the stub is stored in the app (in the `new` folder) and is of course immediately searchable and findable. When we edit and merge we overwrite the stub and synchronize the app and our GitHub repository. If we do not merge, the GitHub does not know that the record exists but it is still there in the app. If you do the merge to another folder then it will not be overwritten but added (resulting in error).

File: new/PRS14536JLowe.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-08-02">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName><forename>J.</forename> <forename>St.</forename> <surname>Lowe</surname></persName>
```
Comment (eu-genia) [inline new/PRS14536JLowe.xml:48]: Is there really a dot after St.? I think `St Lowe` is one surname

Comment (eu-genia) [reply new/PRS14536JLowe.xml:48]: (Can also be spelled `St. Lowe`, it is still one rather well-known surname)

Comment (CarstenHoffmannMarburg) [reply new/PRS14536JLowe.xml:48]: I will change to St. Lowe as a surname.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**